src/app/modules/did-book/components/add-single-record/add-single-record.component.spec.ts

```diff
@@ -11,19 +11,17 @@ describe('AddSingleRecordComponent', () => {
   let fixture: ComponentFixture<AddSingleRecordComponent>;
   const didBookServiceSpy = jasmine.createSpyObj(DidBookService, ['add']);
 
-  beforeEach(
-    waitForAsync(() => {
-      TestBed.configureTestingModule({
-        declarations: [AddSingleRecordComponent],
-        providers: [
-          { provide: DidBookService, useValue: didBookServiceSpy },
-          { provide: MatDialogRef, useValue: dialogSpy },
-          { provide: MAT_DIALOG_DATA, useValue: { did: 'did' } },
-        ],
-        schemas: [NO_ERRORS_SCHEMA],
-      }).compileComponents();
-    })
-  );
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [AddSingleRecordComponent],
+      providers: [
+        { provide: DidBookService, useValue: didBookServiceSpy },
+        { provide: MatDialogRef, useValue: dialogSpy },
+        { provide: MAT_DIALOG_DATA, useValue: { did: 'did' } },
+      ],
+      schemas: [NO_ERRORS_SCHEMA],
+    }).compileComponents();
+  }));
```

Comment:
It seems like most of the files are changed because of a code style change related to new lines for brackets. Is this style change relevant to the "user idle" change?

Reply:
Yeah, i saw it also. Prettier starts failing for now reason. I did not update anything connected to it. It was also failing on GHA.

Reply:
https://github.com/energywebfoundation/switchboard-dapp/runs/7914323996?check_suite_focus=true this one was failing. After changing all this, it is passing. Without this changes in spec files it was also failing on my local version.

Reply:
Ah okay, I see